stripe/src/main/java/com/stripe/android/net/CardParser.java

```diff
@@ -0,0 +1,74 @@
+package com.stripe.android.net;
+
+import android.support.annotation.NonNull;
+
+import com.stripe.android.model.Card;
+import com.stripe.android.util.StripeJsonUtils;
+import com.stripe.android.util.StripeTextUtils;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * A helper class for parsing {@link Card} objects returned from the server.
+ */
+public class CardParser {
+
+    static final String FIELD_ADDRESS_CITY = "address_city";
+    static final String FIELD_ADDRESS_COUNTRY = "address_country";
+    static final String FIELD_ADDRESS_LINE1 = "address_line1";
+    static final String FIELD_ADDRESS_LINE2 = "address_line2";
+    static final String FIELD_ADDRESS_STATE = "address_state";
+    static final String FIELD_ADDRESS_ZIP = "address_zip";
+    static final String FIELD_BRAND = "brand";
+    static final String FIELD_COUNTRY = "country";
+    static final String FIELD_EXP_MONTH = "exp_month";
+    static final String FIELD_EXP_YEAR = "exp_year";
+    static final String FIELD_FUNDING = "funding";
+    static final String FIELD_LAST4 = "last4";
+    static final String FIELD_FINGERPRINT = "fingerprint";
+    static final String FIELD_NAME = "name";
+    static final String FIELD_CURRENCY = "currency";
```

Comment:
Minor nit, but I tend to like alphabetizing this sort of big list because it makes it easier to spot a field and very obvious where to put any new additions.